jobtracker.go

```diff
@@ -42,8 +42,12 @@ const (
 )
 
 var countersToKeep = map[string]string{
-	"FileSystemCounter.HDFS_BYTES_READ":    "hdfs.bytes_read",
-	"FileSystemCounter.HDFS_BYTES_WRITTEN": "hdfs.bytes_written",
+	"FileSystemCounter.HDFS_BYTES_READ":    "bytes_read",
+	"FileSystemCounter.S3_BYTES_READ":      "bytes_read",
+	"FileSystemCounter.FILE_BYTES_READ":	"bytes_read",
+	"FileSystemCounter.HDFS_BYTES_WRITTEN": "bytes_written",
+	"FileSystemCounter.S3_BYTES_WRITTEN":   "bytes_written",
+	"FileSystemCounter.FILE_BYTES_WRITTEN": "bytes_written",
```

Comment:
Cool. I split them out and added them to the title, so people can see the splits. Should be easy to remove FILE if it ends up throwing off numbers.